src/Sulu/Bundle/AdminBundle/Resources/js/components/BlockCollection/types.js

```diff
@@ -8,3 +8,5 @@ export type BlockActionConfig = BlockButtonActionConfig | DividerActionConfig ;
 
 export type RenderBlockContentCallback<T: string, U: {type: T}>
     = (value: U, type: T, index: number, expanded: boolean) => Node;
+
+export type BlockMode = 'static' | 'sortable' | 'selection';
```

Comment:
To unify those modes a bit, we should either go with (`sortable` and `selectable`) or (`sorting` and `selection`), would prefer the first example

Reply:
Used `selectable`